arcade/resources/__init__.py

```diff
@@ -230,14 +237,36 @@ def load_kenney_fonts() -> None:
     """
     from arcade.text import load_font
 
-    load_font(":system:fonts/ttf/Kenney_Blocks.ttf")
-    load_font(":system:fonts/ttf/Kenney_Future.ttf")
-    load_font(":system:fonts/ttf/Kenney_Future_Narrow.ttf")
-    load_font(":system:fonts/ttf/Kenney_High.ttf")
-    load_font(":system:fonts/ttf/Kenney_High_Square.ttf")
-    load_font(":system:fonts/ttf/Kenney_Mini.ttf")
-    load_font(":system:fonts/ttf/Kenney_Mini_Square.ttf")
-    load_font(":system:fonts/ttf/Kenney_Pixel.ttf")
-    load_font(":system:fonts/ttf/Kenney_Pixel_Square.ttf")
-    load_font(":system:fonts/ttf/Kenney_Rocket.ttf")
-    load_font(":system:fonts/ttf/Kenney_Rocket_Square.ttf")
+    load_font(":system:fonts/ttf/Kenney/Kenney_Blocks.ttf")
+    load_font(":system:fonts/ttf/Kenney/Kenney_Future.ttf")
+    load_font(":system:fonts/ttf/Kenney/Kenney_Future_Narrow.ttf")
+    load_font(":system:fonts/ttf/Kenney/Kenney_High.ttf")
+    load_font(":system:fonts/ttf/Kenney/Kenney_High_Square.ttf")
+    load_font(":system:fonts/ttf/Kenney/Kenney_Mini.ttf")
+    load_font(":system:fonts/ttf/Kenney/Kenney_Mini_Square.ttf")
+    load_font(":system:fonts/ttf/Kenney/Kenney_Pixel.ttf")
+    load_font(":system:fonts/ttf/Kenney/Kenney_Pixel_Square.ttf")
+    load_font(":system:fonts/ttf/Kenney/Kenney_Rocket.ttf")
+    load_font(":system:fonts/ttf/Kenney/Kenney_Rocket_Square.ttf")
+
+
+def load_liberation_fonts() -> None:
+    """Loads generic versions of Arial, Courier, and Times New Roman.
+
+    The Liberation font family is a permissively-licensed set of built-ins
+    which should be layout-compatible with popular fonts.
```

Comment:
Can we add the names, which should be used to reference the loaded fonts? That was an issue for me with the Kenny fonts.

Reply:
Yep. Will do.

Reply:
I've added it here as requested, but I believe it may go best on the resources page:
![image](https://github.com/user-attachments/assets/5442ae0b-18d9-4a3e-9e97-900cc9273404)